bot/bot.py

```
@@ -111,22 +111,19 @@ async def register_user_if_not_exists(update: Update, context: CallbackContext,
-     else:
-         return False
+    try:
+        message = update.message
+        if message.chat.type == "private":
+            return True
+        if message.text is not None and ("@" + context.bot.username) in message.text:
+            return True
+        if message.reply_to_message is not None:
+            if message.reply_to_message.from_user.id == context.bot.id:
+                return True
+    except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)